scripts/bert/conversion_tools/convert_pytorch_transformers.py

```diff
@@ -0,0 +1,211 @@
+# coding: utf-8
```

Comment:
How do you intend to maintain and verify this script?

Reply:
They're currently not maintained, and not documented. Ideally we should test them in CI (adding fairseq, tf, pytorch transformer). But I don't have the bandwidth to do that